src/main/resources/languages.yml

```
@@ -65,4 +65,4 @@ custom_5:
   itemModel: ""
 
 # NOTE: Do not change
-config-version: 1.0
\ No newline at end of file
+config-version: 1.0
```

```suggestion
config-version: 1.1
```
bump version so the file will get updated